### PR TITLE
add fixer for price rates

### PR DESCRIPTION
**Pull request by Pantani**
Add currency rates provider from [Fixer](https://fixer.io/) service

### CODE REVIEW

File: pkg/blockatlas/marketdata.go [hunk 1]
```diff
@@ -35,24 +35,24 @@ type TickerPrice struct {
 }
 
 type Rate struct {
```
Comment (madcake) [inline pkg/blockatlas/marketdata.go]: Where is it used?

Comment (Pantani) [reply pkg/blockatlas/marketdata.go]: it's used by storage, market data, and API package 

Screen Shot 2019-11-13 at 14 34 31

Comment (madcake) [reply pkg/blockatlas/marketdata.go]: Not for api result?

Comment (Pantani) [reply pkg/blockatlas/marketdata.go]: no, but we can create a route to retrieve the rate as well, now we are using inside the API package only to apply the currency rate in API result

**Pull request merged**